README.md

```diff
@@ -1,8 +1,97 @@
-# hackshare
+# HackShare
 
-### How to Use
+A web app where fellows can teach other fellows the skills they believe they are good at.
 
-- Download the Repo to your local drive
-- Open `client` in terminal and type `npm start`
-- Open `server` in terminal and type `npm start`
-- Visit `localhost:3000`
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+- [NodeJs](https://nodejs.org/)
+- [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)
+
```

Comment:
Could we just replace the heading as **Requirements** and then just list the tools?